docs/UserGuide.adoc

```diff
@@ -49,9 +62,9 @@ the application.
 
 +
 image::Ui.png[width="790"]
-This is what the GUI looks like when the TA-Tracker is opened.
+This is how the GUI may look like when the TA-Tracker is opened.
 +
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
+.  Type a command in the command box (the box with the `green` words), and press kbd:[Enter] to execute it. +
```

Comment:
Is the statement in the bracket necessary? I can just show what command box is in the diagram... saying green words could be misunderstood to mean that the words always have to be green

Reply:
Sure. Definitely needs a diagram